_maps/map_files/generic/z2.dmm

```diff
@@ -297,23 +301,29 @@
 	},
 /obj/item/weapon/circular_saw,
 /turf/open/floor/holofloor{
-	icon_state = "warnwhite";
+	tag = "icon-white_warn (NORTH)";
+	icon = 'icons/turf/floors/warning.dmi';
+	icon_state = "white_warn";
 	dir = 1
 	},
 /area/holodeck/rec_center/medical)
 "bb" = (
 /obj/structure/table/glass,
 /obj/item/weapon/retractor,
 /turf/open/floor/holofloor{
-	icon_state = "warnwhite";
+	tag = "icon-white_warn (NORTH)";
+	icon = 'icons/turf/floors/warning.dmi';
+	icon_state = "white_warn";
 	dir = 1
 	},
 /area/holodeck/rec_center/medical)
 "bc" = (
 /obj/structure/table/glass,
 /obj/item/weapon/cautery,
 /turf/open/floor/holofloor{
-	icon_state = "warnwhite";
+	tag = "icon-white_warn (NORTHEAST)";
+	icon = 'icons/turf/floors/warning.dmi';
+	icon_state = "white_warn";
```

Comment:
It's still around. Did you edit this or what happened? WJ had this in his lavaland edit as well.